src/components/messenger/list/rewards-tooltip/container.tsx

```diff
@@ -0,0 +1,44 @@
+import * as React from 'react';
+
+import { connectContainer } from '../../../../store/redux-container';
+import { RootState } from '../../../../store/reducer';
+import { rewardsTooltipClosed } from '../../../../store/rewards';
+
+import { RewardsTooltip } from '.';
+import { Container as RewardsItemContainer } from '../../../settings-menu/rewards-item/container';
+export interface PublicProperties {}
+
+export interface Properties extends PublicProperties {
+  meowPreviousDayInUSD: string;
+  isLoading: boolean;
+
+  rewardsTooltipClosed: () => void;
+}
+
+export class Container extends React.Component<Properties> {
+  static mapState(state: RootState): Partial<Properties> {
+    const { rewards } = state;
+    return {
+      meowPreviousDayInUSD: RewardsItemContainer.totalPriceInUSD(rewards.meowPreviousDay, rewards.meowInUSD),
+      isLoading: rewards.loading,
+    };
+  }
+
+  static mapActions(_props: Properties): Partial<Properties> {
+    return { rewardsTooltipClosed };
+  }
+
+  render() {
+    if (this.props.isLoading || this.props.meowPreviousDayInUSD === '$0.00') {
```

Comment:
Probably shouldn't do a comparison based on a formatted string. If we need to know the actual numeric value then we should send it in here as a prop.

Reply:
The `meowPreviousDayInUSD` here is getting `$0.00` set here 
<img width="901" alt="image" src="https://github.com/zer0-os/zOS/assets/33264364/c030e22e-d587-46b0-8493-e28bfcca7669">

This is because it's taking a while to set the "current meow token price". Until then it is set in the state as `0`. Which is because the rewards are coming out to be `0.00`. 

I'll update this check to be against the `meowTokenPrice`.